happypanda/app_constants.py

```diff
@@ -9,44 +10,46 @@
 #GNU General Public License for more details.
 #You should have received a copy of the GNU General Public License
 #along with Happypanda.  If not, see <http://www.gnu.org/licenses/>.
+#"""
 
 """Contains constants to be used by several modules"""
 
-import os, sys, enum
+import enum
+import os
+import sys
+
 import qtawesome as qta
 
-try:
-    import settings
-    from database import db_constants
-except ImportError:
-    from . import settings
-    from .database import db_constants
+from happypanda import settings
+from happypanda.database.arguments import args
+from happypanda.database import db_constants
 
 # Version number
 vs  = '1.0'
 DEBUG = False
 
-OS_NAME = ''
 if sys.platform.startswith('darwin'):
-	OS_NAME = "darwin"
+    OS_NAME = "darwin"
 elif os.name == 'nt':
-	OS_NAME = "windows"
+    OS_NAME = "windows"
+    import ctypes
```

Comment:
This has been done multiple times throughout the codebase already long before I entered the project ;)

I will however move it to the top during the code cleanup PR along with all the others.